### ISSUE TITLE
"smart" code evaluation ? (Ctrl+Return)

### ISSUE THREAD

**Issue by frantic0** [crucialfelix/atom-supercollider#84]
Is there such a feature planned ?

"Normally, the current line will be evaluated (where the cursor is).
If the cursor is within a region, the entire region will be evaluated. See below for explanation of regions.
If some code is selected, the selection will be evaluated."

Many thanks

**Comment by crucialfelix** [comment]
Yes, it's kind of part of this: https://github.com/crucialfelix/atom-supercollider/issues/10

which is smart bracket detection. Then It can detect the region when you evaluate it. It's one of the most important ones to do for usability I think.
